qa/automation/CareerPortal.Tests.Scenarios/2. Extended example/homepage/name_field_validation/email_field_validation/Send_cv/AtachCv.feature

```diff
@@ -0,0 +1,131 @@
+Feature: 1.1.3 Attach CV and Send 
+
+Background: 
+
+	Given User  created 7 byte  "cv1.pdf" file
```

Comment:
All tested files should be placed in your PR.
These files should be prepared by your side. 
Not necessary to list all files in the Background. 
You should use prepared fils in your test cases in this way: 

```
Background: 
Given ... career portal is opened (link should be provided) 

Scenario Outline: 1.1. Send CV form
When the user inputs "Name Name" to the field "Name"
And user inputs "name@mail.ru" to the field "Email"
And user attaches file <File> from the qa/automation/CareerPortal.Tests.Scenarios/1.1. Send CV form/files/
And user clicks on "Send" button
Then CV form should be sent
And user should see the successful message "provide text message" 

Examples: 
|File|
|File1.pdf|
|File 2. txt| 
etc. 

```